Podcast/FeedTableViewCell.swift

```diff
@@ -0,0 +1,374 @@
+        descriptionLabel.frame = CGRect(x: descriptionLabelX, y: descriptionLabelY, width: frame.width - descriptionLabelRightX - descriptionLabelX, height: descriptionLabelHeight)
+        tagsLabel.frame = CGRect(x: tagsLabelX, y: tagsLabelY, width: frame.width - tagsLabelRightX - tagsLabelX, height: tagsLabelHeight)
+        podcastImage.frame = CGRect(x: podcastImageX, y: podcastImageY, width: podcastImageSize, height: podcastImageSize)
+        
+        recommendedLabel.sizeToFit()
+        bookmarkButton.frame = CGRect(x: bookmarkButtonX, y: 0, width: bookmarkButtonWidth, height: bookmarkButtonHeight)
+        recommendedButton.frame = CGRect(x: recommendedButtonX, y: 0, width: recommendedButtonWidth, height: recommendedButtonHeight)
+        recommendedButton.center.y = bottomViewHeight / 2
+        bookmarkButton.center.y = bottomViewHeight / 2
+        playButton.frame = CGRect(x: playButtonX, y: 0, width: playButtonWidth, height: playButtonHeight)
+        moreButton.frame = CGRect(x: moreButtonX, y: 0, width: moreButtonWidth, height: moreButtonHeight)
+        playButton.center.y = bottomViewHeight / 2
+        moreButton.center.y = bottomViewHeight / 2
+        recommendedLabel.center.y = bottomViewHeight / 2
+        recommendedLabel.frame.origin.x = recommendedButton.frame.origin.x + buttonPadding + recommendedButton.frame.width
+        playLabel.frame = CGRect(x: playButtonX + playButtonWidth + buttonPadding, y: 0, width: 0, height: 0)
+        playLabel.sizeToFit()
+        playLabel.center.y = bottomViewHeight / 2
+        
+        feedControlButton.frame = CGRect(x: feedControlButtonX, y: 0, width: feedControlButtonWidth, height: feedControlButtonHieght)
+        feedControlButton.center.y = contextViewHeight / 2
+
+        lineSeperator.frame = CGRect(x: lineSeperatorX, y: mainViewHeight - 1, width: frame.width - 2 * lineSeperatorX, height: lineSeperatorHeight)
+        topLineSeperator.frame = CGRect(x: 0, y: 0, width: frame.width, height: lineSeperatorHeight)
+        seperator.frame = CGRect(x: 0, y: frame.height - seperatorHeight, width: frame.width, height: seperatorHeight)
+        
+        
+    }
+    
+
+    ///
+    ///Mark - Buttons
+    ///
+    func bookmarkButtonPress() {
+        bookMarked = !bookMarked
+        if bookMarked {
+            bookmarkButton.setImage(#imageLiteral(resourceName: "bookmarkFeedIcon"), for: .normal)
+        } else {
+            bookmarkButton.setImage(#imageLiteral(resourceName: "bookmarkFeedIcon_unselected"), for: .normal)
+        }
+        
+    }
+    
+    func recommendedButtonPress() {
+        recommended = !recommended
+        if recommended {
+            recommendedButton.setImage(#imageLiteral(resourceName: "heart_icon_selected"), for: .normal)
+        } else {
+            recommendedButton.setImage(#imageLiteral(resourceName: "heart_icon"), for: .normal)
+        }
+    }
+    
+    func moreButtonPress() {
+        
+    }
+    
+    func playButtonPress() {
```

Comment:
I'm not sure how the player is implemented, but I guess we'd want the view controller to be told that the play button was pressed (same for the more button). Probably doesn't need to be in this PR but a delegate for this class might be helpful